### PR TITLE
[ENG-1507] More documentation

### PR DESCRIPTION
**Pull request by jamiepine**
This PR contains a new blog post, updated roadmap, more docs and landing tweaks.

Screenshot 2024-02-06 at 4 01 35 AM

## Summary by CodeRabbit

- **New Features**
	- Introduced drag and drop functionality, AI labeling for images, advanced search capabilities, tab support, multi-language support, and an updated roadmap.
	- Added new features including device connections, iOS and Android apps, and column view for file management.
	- Launched a React component for embedding responsive videos.
	- Unveiled the "Spacedrive Integrity and Innovation Pledge" and focused narrative on Spacedrive's commitment to longevity.
	- Introduced guides and documentation on Virtual Distributed Filesystem (VDFS), Actions, Albums, Automations, Cloud Support, Comments, Database Synchronization, Duplicates, Folder Sync, Habits, Inspector, Interface, Jobs, Library Setup, Locations, Media Conversion, Media View, Models, Nodes, People, Preview Media, Quick Actions, Quick View, Search, Spacedrop, Spaces, Tags, and ongoing security enhancements.

- **Documentation**
	- Enhanced and updated various guides and documentation to provide clearer instructions and information on Spacedrive features and functionalities.

- **Style and User Experience Improvements**
	- Adjusted CSS classes for styling elements like images, headings, paragraphs, and tags across the blog and roadmap pages.
	- Added animations and transitions for a dynamic user experience.
	- Improved image width in blog posts for better visual presentation.
	- Simplified class assignment logic in the roadmap module for cleaner code.

- **Roadmap and Future Plans**
	- Updated the release timeline to Q4 2024 and included detailed descriptions of upcoming features and enhancements in the roadmap.

### CODE REVIEW

File: apps/landing/posts/drag-and-drop.mdx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+It's been an exciting couple of months since we launched Spacedrive. We're thrilled to share that we've hit **139,000** unique installations, with an average runtime of **50 minutes**. The enthusiasm and feedback from our community have been incredible, and your requests for new features have not gone unnoticed.
+
+Our team has been working tirelessly to turn your most-wanted features into reality. Let's take a look at what's new in v0.2...
+
+## 🫳 Drag and drop
+
+<Video url="/videos/Spacedrive_DragAndDrop.webm" />
+
+The Explorer is the most intricate user interface we have ever designed, now with drag and drop it is one big step closer to having everything you need from a file explorer. In an upcoming release, we'll be adding support for dragging files and folders into and out of Spacedrive from the OS.
```
Comment (PineappleRind) [inline apps/landing/posts/drag-and-drop.mdx:18]:
```suggestion
The Explorer is the most intricate user interface we have ever designed. Now, with drag and drop, it is one big step closer to having everything you need from a file explorer. In an upcoming release, we'll be adding support for dragging files and folders into and out of Spacedrive from the OS.
```

File: docs/product/guides/actions.mdx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+Actions in Spacedrive signify the user's intention to modify the filesystem, often with the possibility of undoing the change. These actions may result in an immediate operation or queue a job composed of one or more tasks.
+
+Spacedrive allows actions to be executed by remote Nodes, optimizing the process based on available resources. For instance, if an action involves converting a MOV file to MP4 and the file exists on another device, the device containing the file will perform the conversion. This decision also takes into account factors like available CPU power, aiming to select the best option automatically.
+
+While Spacedrive strives to make intelligent _default_ choices for action execution, users retain the flexibility to override these decisions if desired. The system is designed to minimize user intervention while maintaining user control.
+
+## Using natural language
+
+Spacedrive actions can be created using natural language, by pressing CMD/CTRL + K you can describe a desired action, for example "Move all screenshots from my Desktop onto my NAS". Spacedrive will use all available data sources to best determine a planned list of actions based on your query.
```
Comment (PineappleRind) [inline docs/product/guides/actions.mdx:17]:
```suggestion
By pressing CMD/CTRL + K you can describe a desired action using natural language, for example "Move all screenshots from my Desktop onto my NAS". Spacedrive will use all available data sources to best determine a planned sequence of actions based on your query.
```

Comment (brxken128) [reply docs/product/guides/actions.mdx:17]: I tweaked this a little, and went with `By pressing CMD/CTRL + K, you can describe a desired Spacedrive action using natural language, for example "Move all screenshots from my Desktop onto my NAS". Spacedrive will use all available data sources to best determine a planned sequence of actions based on your query.
`. Thank you!

File: apps/landing/posts/drag-and-drop.mdx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+## Tabs
+
+<Video url="/videos/Spacedrive_Tabs.webm" />
+
+You can open multiple tabs in the Explorer, and switch between them using the tab bar at the top of the window.
+
+## Now in 12 languages!
+
+We've expanded our language options to include English, Japanese, Chinese (Simplified and Traditional), German, French, Hindi, Portuguese, Dutch, Italian, Russian, Korean, Spanish, Turkish, and Arabic. Some translations were created with AI assistance, so feel free to submit corrections on [GitHub](https://github.com/spacedriveapp/spacedrive).
```
Comment (HeavenVolkoff) [inline apps/landing/posts/drag-and-drop.mdx:38]: We still don't support:
 - Japanese
 - Hindi
 - Portuguese
 - Korean
 - Arabic
 
 See: https://github.com/spacedriveapp/spacedrive/blob/69a6edb99f2d1b421914ffd1e100c4f4a70d9af1/interface/app/%24libraryId/settings/client/general.tsx#L27-L39

File: apps/landing/posts/drag-and-drop.mdx [hunk 4]
```diff
@@ -0,0 +1,68 @@
+---
+author: Jamie Pine
+title: Introducing Spacedrive Alpha v0.2 - Drag & drop, AI Labels, Advanced Search, 12 Languages, Tabs,  and Spacedrop!
```
Comment (HeavenVolkoff) [inline apps/landing/posts/drag-and-drop.mdx:3]: 11 Languages

File: apps/landing/posts/drag-and-drop.mdx [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+
+## And so much more...
+
+We've also shipped these major features since the launch:
+
+- Multi-select in the Explorer
+- A bar for Quick View to view surrounding media
+- CPU usage control slider
+- Improved virtual filesystem
+- Way. More. Docs. [See Docs →]()
```
Comment (HeavenVolkoff) [inline apps/landing/posts/drag-and-drop.mdx:54]: Needs link to docs

File: apps/landing/posts/drag-and-drop.mdx [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+
+## Now in 12 languages!
+
+We've expanded our language options to include English, Japanese, Chinese (Simplified and Traditional), German, French, Hindi, Portuguese, Dutch, Italian, Russian, Korean, Spanish, Turkish, and Arabic. Some translations were created with AI assistance, so feel free to submit corrections on [GitHub](https://github.com/spacedriveapp/spacedrive).
+
+## Updated roadmap
+
+![Spacedrive Roadmap](/images/roadmap.jpg)
+
+We've updated our roadmap to reflect the progress we've made since launch, and to give you a better idea of what's coming next. From today, we are aiming for a monthly major release cycle, with minor releases in between. Check out the [Roadmap →]()
```
Comment (HeavenVolkoff) [inline apps/landing/posts/drag-and-drop.mdx:44]: Needs link to roadmap

**Review by HeavenVolkoff — CHANGES_REQUESTED** [review]

File: apps/landing/posts/drag-and-drop.mdx (hunk 3 above)
Comment (brxken128) [reply apps/landing/posts/drag-and-drop.mdx:38]: Arabic is a tricky one too, as there's _so many_ different dialects. There's MSA (Modern Standard Arabic) but then almost every Arabic-speaking country [has a dialect of their own](https://en.wikipedia.org/wiki/Varieties_of_Arabic#Classification) which differs from MSA. That _is_ a very verbose list, they can be grouped up a little bit more, but regardless; many lack mutual-intelligibility as far as I'm aware, and we'd need to support MSA as well as many of the dialects.

Comment (brxken128) [reply apps/landing/posts/drag-and-drop.mdx:38]: On another note, we could probably add these languages using AI. I'm not sure how @utkubakir has been handling them, I tried but ChatGPT just kept making me hit continue writing and eventually gave up.

File: apps/landing/posts/drag-and-drop.mdx [hunk 7]
```diff
@@ -0,0 +1,68 @@
+---
+author: Jamie Pine
+title: Introducing Spacedrive Alpha v0.2 - Drag & drop, AI Labels, Advanced Search, 12 Languages, Tabs,  and Spacedrop!
+tags: [News, Updates, Release]
+date: 2023-12-14
```
Comment (brxken128) [inline apps/landing/posts/drag-and-drop.mdx:5]: Date is massively behind for a Feb. release

File: apps/landing/posts/drag-and-drop.mdx (hunk 3 above)
Comment (utkubakir) [reply apps/landing/posts/drag-and-drop.mdx:38]: Let's just remove what we don't support atm

**Review by utkubakir — APPROVED** [review]

**Review by utkubakir — APPROVED** [review]

**Pull request merged**